packages/react-native-bpk-component-calendar/README.md

```diff
@@ -259,3 +263,30 @@ An any matcher will match if the date is equal to any of the dates provided.
 ```
 
 Returns **DateMatcher** an any date matcher.
+
+### colorBucket
+
+Creates a new color bucket to be used in BpkCalendar.
+
+A color bucket is used to define custom colours for calendar days.
+
+#### Parameters
+
+-   `color` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** The background color
+-   `days` **DateMatcher** The days in this bucket
```

Comment:
What happens if you have overlapping DateMatchers?

Reply:
Last one takes precedence, meant to write that down but forgot